src/registered.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const fs = require('fs-extra');
+const low = require('lowdb');
+const FileSync = require('lowdb/adapters/FileSync');
+const amministrazioni = require('../public/assets/data/authorities.db.json');
+
+const whitelistFile = 'private/data/whitelist.db.json';
+fs.ensureFileSync(whitelistFile);
+const adapter = new FileSync(whitelistFile);
+const db = low(adapter);
+
+// Set some defaults (required if your JSON file is empty)
+db.defaults({ registrati: [] }).write()
+
+module.exports = function(request, reply){
+    const referente = request.query.referente;
+    const ipa = request.query.ipa;
+    const url = request.query.url;
+    const pec = amministrazioni[ipa].pec;
+
+    db.get('registrati')
+    .push({
+        referente: referente,  
+        ipa: ipa,
+        url: url,
+        pec: pec
+    })
+    .write()
+
+    return 'Registrazione avvenuta con successo';
```

Comment:
Ti ricordo che in questa pagina serve una ulteriore maschera di conferma che riporti nuovamente le informazioni che si sta validando e un tasto "Conferma". Poiché l'email girerà per gli uffici, vorrei evitare che qualcuno clicchi anche solo per curiosità e così facendo confermi subito la registrazione.

Reply:
Si si l'ho messo in conto. Lo includo nella prossima PR.